react-context/flow.txt

```diff
@@ -18,8 +18,13 @@
 
 ## User can sign in
   When the user 'landed' on the home page
-    It will check if the user is logged in through the use of tokens
-    If the user is not logged in it will take the user to the 'sign-in' page
+  After the first render of the home page, the client call the componentDidMount method
+    It will check if the user is logged in through the use of tokens in the local storage
+    If there is a token, the client then parse the token into a user object
+    If there is no token, the client will set the user object to null
+    the the client would re-render the page based on the value of the user object
+    If there is a user the client automatically login and the client render the home page of the app
+    If the the User object is null the client render the auth forms with the sign-in path
```

Comment:
You are getting much closer, but how does the app "automatically" show home or sign in? Hint, look for `<Redirect />`